release-tools/check-crate-version/src/main.rs

```diff
@@ -0,0 +1,177 @@
+use std::{cmp::max, collections::HashMap, fs, path::PathBuf, process::Command};
+
+use clap::Parser;
+use serde::Deserialize;
+use toml_diff::TomlDiff;
+
+mod download;
+
+use download::download_crate;
+
+const PUBLISH_LIST_PATH: &str = "./publish-list.toml";
+const CRATES_DIR: &str = "../../crates";
+const CRATES_IO_DIR: &str = "./crates_io";
```

Comment:
These paths can keep those default values, but they should be overridable via CLI flags and environment variables. I believe Clap will handle both.

Reply:
Implemented